marklogic-data-hub-central/ui/src/App.scss

```diff
@@ -119,6 +119,14 @@ tr.ant-table-row:hover > td {
   content: "\22C1";
 }
 
+.ant-table-body {
+  overflow: auto !important;
+}
+
+.ant-table-header {
```

Comment:
This may override the style of all tables. Can you please confirm UX change with Marilyn.

Reply:
Confirmed with @timur-isangulov nothing changes with tables across the app, but solves scrollbar display problems on Windows.